lms/templates/support/program_enrollments_inspector.html

```diff
@@ -0,0 +1,37 @@
+<%page expression_filter="h"/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.js_utils import js_escaped_string
+%>
+
+## Override the default styles_version to use Bootstrap
+<%! main_css = "css/bootstrap/lms-main.css" %>
+
+<%namespace name='static' file='../static_content.html'/>
+
+<%inherit file="../main.html" />
+
+<%block name="js_extra">
+</%block>
+
+<%block name="pagetitle">
+	${_("Program Enrollments Inspector")}
+</%block>
+
+<%block name="content">
+	<section class="container outside-app">
+        <h3> Program Enrollments Inspector </h3>
+            ${static.renderReact(
```

Comment:
Extremely high-level and maybe not-as-actionable feedback: leveraging react adds substantially to the time it takes for the page to be ready and interactable, so the choice to use it here (while probably just motivated by it being the recommended frontend technology) doesn't jive super well with a "reload the page every time the user does something" set-up. So ideally we'd move to ajax-based updates so we don't suffer a loss of usability by being current.